pkg/config/config.go

```diff
@@ -25,8 +25,9 @@ import (
 )
 
 var defaultReplicaConfig = &ReplicaConfig{
-	CaseSensitive:  true,
-	EnableOldValue: false,
+	CaseSensitive:    true,
+	EnableOldValue:   false,
+	CheckGCSafePoint: true,
```

Comment:
Why add flag? I think it must be checked all the time.

Reply:
cause of the bug in pd (tikv/pd#3128), if TiCDC(4.0.9+) try to connect to a TiKV(which of the version is less then 4.0.8), the gc safe point check will failed. so we add a flag to skip the check

Reply:
What about checking GC safepoint by cluster version automatically?

Reply:
It's not strict I think. the version check is not correct all the time(for example : the version of a binary built in the master branch may be 4.0.x-xxx-dirty)

Reply:
and multi-version components in one cluster are not recommended for production. I think it's enough that we just make sure that the cdc is able to connect to the old version tikv